### PR TITLE
feat: add global_inflation_per_claim param to tokenomics module

### PR DESCRIPTION
**Pull request by bryanchriswhite**
## Summary

Add `global_inflation_per_claim` param to the tokenomics module.

## Issue

- `TODO_BETA`

## Type of change

Select one or more from the following:

- [x] New feature, functionality or library
- [ ] Consensus breaking; add the `consensus-breaking` label if so. See #791 for details
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Documentation**: `make docusaurus_start`; only needed if you make doc changes
- [x] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [x] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [ ] I have commented my code
- [x] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

File: tools/scripts/params/tokenomics_dao_reward_address.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/poktroll.tokenomics.MsgUpdateParam",
+        "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+        "name": "mint_allocation_percentages",
```
Comment (bryanchriswhite) [inline tools/scripts/params/tokenomics_dao_reward_address.json:7]:
```suggestion
        "name": "global_inflation_per_claim",
```

File: tools/scripts/params/tokenomics_global_inflation_per_claim.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/poktroll.tokenomics.MsgUpdateParam",
+        "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+        "name": "mint_allocation_percentages",
```
Comment (bryanchriswhite) [inline tools/scripts/params/tokenomics_global_inflation_per_claim.json:7]:
```suggestion
        "name": "global_inflation_per_claim",
```

File: x/tokenomics/keeper/msg_update_params_test.go [hunk 3]
```diff
@@ -70,6 +70,32 @@ func TestMsgUpdateParams(t *testing.T) {
 			shouldError:    true,
 			expectedErrMsg: "empty address string is not allowed",
 		},
+		{
+			desc: "invalid: negative global inflation per claim",
+
+			req: &tokenomicstypes.MsgUpdateParams{
+				Authority: tokenomicsKeeper.GetAuthority(),
+				Params: tokenomicstypes.Params{
+					GlobalInflationPerClaim: -0.1,
```
Comment (bryanchriswhite) [inline x/tokenomics/keeper/msg_update_params_test.go:79]:
```suggestion
					// GlobalInflationPerClaim MUST be positive.
					GlobalInflationPerClaim: -0.1,
```

File: tools/scripts/params/tokenomics_dao_reward_address.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/poktroll.tokenomics.MsgUpdateParam",
+        "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+        "name": "global_inflation_per_claim",
```
Comment (bryanchriswhite) [inline tools/scripts/params/tokenomics_dao_reward_address.json:7]:
```suggestion
        "name": "dao_reward_address",
```

File: x/tokenomics/types/message_update_param_test.go [hunk 5]
```diff
@@ -63,6 +63,17 @@ func TestMsgUpdateParam_ValidateBasic(t *testing.T) {
 			},
 			expectedErr: ErrTokenomicsParamInvalid,
 		},
+		{
+			name: "invalid: global inflation per claim less than 0",
```
[the remark this answers is not in the document]
Comment (red-0ne) [reply x/tokenomics/types/message_update_param_test.go:67]: `0` is a valid value, giving a non-zero value is to enable it.

**Review by red-0ne — APPROVED** [review]

**Pull request merged**